src/util/xml.cpp

```diff
@@ -240,7 +241,9 @@ std::string xmlt::unescape(const std::string &str)
         result+=c;
       }
       else
-        throw "XML escape code not implemented"; // NOLINT(readability/throw)
+        // TODO review if this should be an invariant e.g: - effectively the
```

Comment:
Not sure, but it looks like this code is meant to be run on unsanitised user input, in which case this should remain an exception.